conditional_register.go

```diff
@@ -0,0 +1,16 @@
+// +build register_crdb
+
+// andrei: Unconditional registration of the pq driver under the name "postgres"
+// was remove from conn.go and added to this conditionally-built file in order
+// to support linking this forked lib/pq with an unforked one. Users of the
+// forked lib don't need to use it through Go's sql package, so they don't need
+// this registration. pq's own tests do, however, so they must be built with
+// this build tag.
```

Comment:
Well I think changing all the tests is too intrusive; I think we should keep our patches as minimal as possible.
Two of the tests are in fact currently failing, so it doesn't look like we cared too much about them... So we could drop this file all-together. But I think it doesn't hurt.